.github/workflows/release.yml

```
@@ -0,0 +1,72 @@
+# This workflow does the following:
+#
+#   1. builds botan on Ubuntu and Windows
+#   2. creates a release, if it doesn't exist
+#   3. uploads the artifacts to that release
+#
+# This is done whenever a new tag is created.
+#
+name: release
+
+on:
+  push:
+    tags:
+      - "*"
```

Ah wait, do we want to change this to only match v3 release tags? e.g.

```suggestion
    tags:
      - "v3*"
```

---

We could, but should we? For now, I do hope that we have only one release flavor of Botan for quite some time, which is 3.

---

Yeah, I tend to agree and hope the same. Then let's keep it as is.